### PR TITLE
fix(scalasdk): Scala PB Any vs Scala Any.

### PR DESCRIPTION
**Pull request by RayRoestenburg**

### CODE REVIEW

File: codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala [hunk 1]
```diff
@@ -113,7 +113,12 @@ object SourceGeneratorUtils {
     override def toString: String = {
       val suffix = if (semi) ";" else ""
       imports
-        .map(pkg => s"import $pkg$suffix")
+        .map { pkg =>
```
Comment (raboof) [inline codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:116]: I guess `pkg` is a bad variable name here, sorry :)

**Review by raboof — APPROVED** [review]
Nice ;)

**Review by octonato — APPROVED** [review]
LGTM, after fixing variable name :-)

File: codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala [hunk 2]
```diff
@@ -113,7 +113,12 @@ object SourceGeneratorUtils {
     override def toString: String = {
       val suffix = if (semi) ";" else ""
       imports
-        .map(pkg => s"import $pkg$suffix")
+        .map { imported =>
+          if (imported == "com.google.protobuf.any.Any") {
+            s"import com.google.protobuf.any.{ Any => ScalaPbAny }$suffix"
```
Comment (patriknw) [inline codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:118]: 👍

Comment (johanandren) [reply codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:118]: Isn't this used by java codegen as well?

Comment (johanandren) [reply codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:118]: I guess that would have failed some test.

Comment (patriknw) [reply codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:118]: In Java it's `com.google.protobuf.Any` (note the difference in `any.Any` at the end.

and in Java `Any` is perfectly fine.

**Pull request merged**